src/basic/random-util.c

```diff
@@ -276,6 +351,38 @@ void pseudo_random_bytes(void *p, size_t n) {
 
 void random_bytes(void *p, size_t n) {
 
+        /* This returns high quality randomness if we can get it cheaply. If we can't because for some reason
+         * it is not available we'll try some crappy fallbacks.
```

Comment:
"because for some reason it is not available" → this contains no informational value.